app/utils/theme.js

```diff
@@ -42,3 +44,7 @@ export function isThemeSwitchingEnabled(state) {
     const {config} = state.entities.general;
     return config.EnableThemeSelection === 'true';
 }
+
+export function getKeyboardAppearanceFromTheme(theme) {
+    return tinyColor(theme.centerChannelBg).isLight() ? 'light' : 'dark';
```

Comment:
Hey @cardoso please add a unit test for this function

Reply:
Nice catch! Done :)